app/main.py

```diff
@@ -1,2 +1,72 @@
+from __future__ import annotations
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 0
+        self.hash_table: list = [None] * 8
+
+    def resize_hash_table(self) -> bool:
+        if self.length >= int(len(self.hash_table) * 2 / 3):
```

Comment:
Consider using a variable, such as `LOAD_FACTOR`, to represent the load factor value (2/3) here

Reply:
👍